homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,29 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    int resultint;
+    std::string result;
+        if (command == "add"){
+            resultint = first + second;
+            result = std::to_string(resultint);
+            }
```

Comment:
The code can be simplified in this way:
```suggestion
        if (command == "add"){
            return std::to_string(first + second);
        }
```